go.mod

```diff
@@ -175,3 +176,5 @@ require (
 	sigs.k8s.io/service-apis v0.0.0-20200625154238-3290dca43e1e
 	sigs.k8s.io/yaml v1.2.0
 )
+
+replace sigs.k8s.io/service-apis => github.com/howardjohn/service-apis v0.0.0-20200625215450-120bc372d5a7
```

Comment:
I have been working on this a long time and am getting impatient so I wouldn't mind getting it committed for now because the review is pretty slow there but thats probably a major bias from me and if I was on the other side I would ask to wait for it to merge. https://github.com/kubernetes-sigs/service-apis/pull/217 is the PR btw

Reply:
I'd wait to merge this PR if its only a few days for 217 to go in. Otherwise, opening a tracking issue and referencing it here might be fine if you're confident the merged PR wouldn't be radically different than what exists now. You could also create a temporary fork under the Istio org if iother's are concerned about the use of personal repos.

Reply:
There is a slim chance it will change, the codegen is pretty rigid. I'll just bug them to review it though